### ISSUE TITLE
Export quantized model

### ISSUE THREAD

**Issue by cattpku** [NervanaSystems/distiller#89]
Hi Neta,

I looked into the doc and find out that in https://nervanasystems.github.io/distiller/design/index.html#quantization, it is mentioned 'We also provide a mechanism which takes an existing model and automatically replaces required operations with quantized versions.' Does this mean we can export this model with such 'quantized versions' operation?
If it is not supported in current Distiller, can you kindly suggest how I can export it?
Thanks so much.

**Comment by cattpku** [comment]
Hi Neta,
I tried to export a non-quantized model to ONNX as advised, however, got a RuntimeError 'ONNX export failed: Couldn't export operator aten::softmax'.

I know this layer is added in Distiller, and I checked the corresponding 'torch/onnx/symbolic.py', the 'softmax' definition is there. Do you have any idea about this error? Thanks.

**Comment by nzmora** [comment]
Hi @cattpku ,

Please share your command-line, so I can try to recreate this problem.  
I think that using 
```
softmax = torch.nn.Softmax(dim=-1)
```
is probably more correct, but I want to see why it fails for you before making changes to the code.

BTW, you can also try to change it to ```-1``` and report if this fixes the fail you see.

Thanks
Neta

**Comment by cattpku** [comment]
Hi Neta,
I am not using command-line, as I put the whole code in Pycharm  to run it. I just set the following args:
--data:  _my own dataset path_
--arch:  _my own network definition_
--summary:  _default = 'onnx'_

I tried to change it to -1 as suggested, it went well, no error occurred.

**Comment by nzmora** [comment]
Thanks, for classification tasks the softmax input has shape==(1, num_classes) so 
```
softmax = torch.nn.Softmax(dim=1)
```
and
```
softmax = torch.nn.Softmax(dim=-1)
```
are the same.  But as you point out, for the general case we should follow the [PyTorch ONNX limitations](https://pytorch.org/docs/stable/onnx.html#supported-operators).
